src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageViewModel.cs

```diff
@@ -284,6 +286,9 @@ public bool IsComplete()
             {
                 return SelectedSpecification != null && SelectedStyle != null && SelectedNotificationPreference != null;
             }
+
+            // For screen readers
+            public override string ToString() => ServicesVSResources.Naming_Rule;
```

Comment:
Why is the capitalization different between `Naming_rules` and `Naming_Rule`?

Reply:
`Naming_Rule` pre-existed and I didn't think it would matter to a screen reader.

Reply:
hmm, and we can't reuse the existing resource?